remote-config/USERGUIDE.md

```diff
@@ -22,10 +22,12 @@ repositories {
 }
 
 dependency {
-  implementation 'com.rakuten.tech.mobile.remoteconfig:remote-config:1.0.0'
+  implementation 'com.rakuten.tech.mobile.remoteconfig:remote-config:${latest_version}'
 }
 ```
 
+Note: please use/enable R8 to avoid proguard issue with Moshi.
```

Comment:
Could you add a little more explanation on how they can do this? Or maybe just adding a link to the docs on R8 is enough: https://developer.android.com/studio/build/shrink-code

Reply:
Added link